src/firecracker/src/main.rs

```diff
@@ -191,7 +191,8 @@ fn main() {
         // It's safe to unwrap here because the field's been provided with a default value.
         let level = arguments.value_as_string("level").unwrap();
         let logger_level = LoggerLevel::from_string(level).unwrap_or_else(|err| {
-            panic!("Invalid value for logger level: {}. Possible values: [Error, Warning, Info, Debug]", err);
+            error!("Invalid value for logger level: {}. Possible values: [Error, Warning, Info, Debug]", err);
+            process::exit(i32::from(vmm::FC_EXIT_CODE_GENERIC_ERROR));
```

Comment:
nit: I see we already use this in 2 places. If we plan to make extended use of it, we can define a macro, just like `panic!`. Something like `fc_exit!(exit_code, message, args)`.

Reply:
I think it would affect readability pretty much (I tried something similar here: https://github.com/firecracker-microvm/firecracker/pull/1945 and the result wasn't so nice) and I also expect the number of code lines won't decrease. I would leave it as is if you don't have a strong preference against it.

Reply:
Ok. I don't have a strong preference.